exo-add/features/add-service.feature

```diff
@@ -0,0 +1,47 @@
+Feature: interactive scaffolding
+
+  As a developer not using ExoService
```

Comment:
> As a tech lead
I want to define the templates my team uses for creating new services
So that they don't copy-and-past old code around.